docs/east_ocr.md

```diff
@@ -29,26 +29,62 @@ The metadata are the `text_coordinates` and the `confidence_level` outputs.
 ### East-resnet50 model
 
 The original pretrained model for east-resnet50 topology is stored on https://github.com/argman/EAST in TensorFlow checkpoint format.
-Download and unzip the fle east_icdar2015_resnet_v1_50_rbox.zip to folder.
+
+Clone github repository:
+```bash
+git clone https://github.com/argman/EAST 
+cd EAST 
+```
+Download and unzip the fle east_icdar2015_resnet_v1_50_rbox.zip to EAST folder with the github repository.
 ```bash
 unzip ./east_icdar2015_resnet_v1_50_rbox.zip
 ```
-Convert the TensorFlow Model to Intermediate Representation format using the model_optimizer tool. It can be done
-using a public docker image with OpenVINO toolkit:
+Inside the EAST folder add a file `freeze_east_model.py` with the following content:
```

Comment:
Maybe instead adding this to .md file add it to separate .py file and refer to this script?